saleor/product/models/base.py

```diff
@@ -156,54 +206,79 @@ class Meta:
         verbose_name_plural = pgettext_lazy('Product model', 'products')
 
     def __iter__(self):
+        # type: () -> Iterable[django.db.models.QuerySet]
```

Comment:
You'd have to import `Iterable` first to use it here. There is no such builtin. Also this signature means "returns an iterable that yields QuerySets" while I'm sure it returns a single QuerySet (which is an `Iterable` of `ProductVariant` instances).